### PR TITLE
[FEATURE] Support rtl mode

### PR DESCRIPTION
**Pull request by klauste**
LRN-28950

### CODE REVIEW

File: classes/Learnosity/Plugin.php [hunk 1]
```diff
@@ -34,28 +39,62 @@ public function enqueue_scripts()
 		$lrn_reports_api_url = get_option('lrn_reports_api_url','https://reports-va.learnosity.com/?v1');
 		$lrn_author_api_url = get_option('lrn_author_api_url','https://authorapi-or.learnosity.com/?v1');
 		wp_enqueue_script(
-			'learnosity-items',
-			$lrn_items_api_url,
-			array(),
-			null,
-			false
+		    'learnosity-items',
+            $lrn_items_api_url,
```
Comment (PhilBroadbent) [inline classes/Learnosity/Plugin.php:50]: I think the indentation change here was checked in by accident? It looked correct before [here](https://github.com/Learnosity/wordpress-learnosity/blob/master/classes/Learnosity/Plugin.php#L36)

File: classes/Learnosity/Plugin.php [hunk 2]
```diff
@@ -34,28 +39,62 @@ public function enqueue_scripts()
...
-			array(),
-			null,
-			false
+		    'learnosity-reports',
+            $lrn_reports_api_url,
+            array(),
+            null,
+		    false
 		);
         wp_enqueue_script(
```
Comment (PhilBroadbent) [inline classes/Learnosity/Plugin.php:62]: Indentation of this function call is wrong, so would be good to fix here thanks

File: classes/Learnosity/Plugin.php [hunk 3]
```diff
@@ -34,28 +39,62 @@ public function enqueue_scripts()
...
-			'learnosity-items',
-			$lrn_items_api_url,
-			array(),
-			null,
-			false
+		    'learnosity-items',
+            $lrn_items_api_url,
+            array(),
+            null,
+            true
```
Comment (PhilBroadbent) [inline classes/Learnosity/Plugin.php:53]: Did you mean to change this `in_footer` variable to true? If so, how come?

File: classes/Learnosity/Plugin.php [hunk 4]
```diff
@@ -34,28 +39,62 @@ public function enqueue_scripts()
...
         wp_enqueue_script(
             'learnosity-author',
             $lrn_author_api_url,
             array(),
             null,
-            false
+            true
+        );
+
+        // Before the script tags are added to page, add the rtl data attribute
```
Comment (PhilBroadbent) [inline classes/Learnosity/Plugin.php:70]: Indentation wrong here, too

File: classes/Learnosity/Plugin.php [hunk 5]
```diff
@@ -34,28 +39,62 @@ public function enqueue_scripts()
...
+        add_filter(
+            'script_loader_tag',
+            array($this, 'add_rtl_data'),
+            10,
+            3
         );
 	}
 
+    /**
+     * Adds data-lrn-dir="rtl" to the source tag if $this->>is_rtl is true
```
Comment (PhilBroadbent) [inline classes/Learnosity/Plugin.php]: Typo `->>` -> `->`

File: classes/Learnosity/Shortcodes/Generator.php [hunk 6]
```diff
@@ -36,6 +41,9 @@ public function render_item($attrs)
     public function render_items($attrs, $content)
     {
         $items_embed = new ItemsEmbed($attrs, 'inline', $content);
+        if (isset($attrs['rtl']) && $attrs['rtl'] === 'true') {
```
Comment (PhilBroadbent) [inline classes/Learnosity/Shortcodes/Generator.php]: Logic is repeated 4 times, ideally extract to function

**Review by PhilBroadbent — CHANGES_REQUESTED** [review]

File: classes/Learnosity/Plugin.php (hunk 1 above)
Comment (klauste) [reply classes/Learnosity/Plugin.php:50]: Indentation before was partially not with spaces, fixed it in new commit

File: classes/Learnosity/Plugin.php (hunk 3 above)
Comment (klauste) [reply classes/Learnosity/Plugin.php:53]: Yes, adding comment:
/*
         * For items and author, in_footer is set to true. This is on purpose as the information, whether
         * the read direction is right to left is set in the plugin short-code. If we loaded the scripts
         * in the header, they'd be loaded before the short-code scripts are encountered, thus we
         * wouldn't know whether to add the rtl flag in the src tag.
         */

**Review by PhilBroadbent — APPROVED** [review]

**Pull request merged**